configure.ac

```diff
@@ -10,7 +10,7 @@ AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_AUX_DIR([config])
 AC_CONFIG_MACRO_DIR([m4])
 
-AM_INIT_AUTOMAKE([foreign tar-pax])
+AM_INIT_AUTOMAKE([foreign tar-pax subdir-objects])
```

Comment:
Yeah, you have a couple options. You could use `git rebase -i maint-1.2` and choose to edit the commit to remove this change.